app/controllers/api/v0/product_images_controller.rb

```diff
@@ -16,9 +16,12 @@ def update_product_image
 
         success_status = image.persisted? ? :ok : :created
 
-        image.update(attachment: params[:file])
-
-        render json: image, serializer: ImageSerializer, status: success_status
+        if image.update(attachment: params[:file])
+          render json: image, serializer: ImageSerializer, status: success_status
+        else
+          error_json = { errors: image.errors.full_messages }
```

Comment:
I'm wondering how this one can be i18ned?

Reply:
It already is:

https://github.com/openfoodfoundation/openfoodnetwork/blob/df6e107dd363bf0aaf9cc18f90cd3f12f39aa204/config/locales/en.yml#L86-L107

This is custom to a validation gem using this. The standard approach would be to add to `en.activerecord.errors` as we do here:

https://github.com/openfoodfoundation/openfoodnetwork/blob/df6e107dd363bf0aaf9cc18f90cd3f12f39aa204/config/locales/en.yml#L62-L84